distributedlog-client/src/main/java/com/twitter/distributedlog/client/serverset/DLZkServerSet.java

```diff
@@ -33,19 +36,43 @@
  */
 public class DLZkServerSet {
 
+    private static final Logger logger = LoggerFactory.getLogger(DLZkServerSet.class);
+
     static final String ZNODE_WRITE_PROXY = ".write_proxy";
 
     private static String getZKServersFromDLUri(URI uri) {
         return uri.getAuthority().replace(";", ",");
     }
 
+    private static Iterable<InetSocketAddress> getZkAddresses(URI uri) {
+        String zkServers = getZKServersFromDLUri(uri);
+        String[] zkServerList = StringUtils.split(zkServers, ',');
+        InetSocketAddress[] zkAddresses = new InetSocketAddress[zkServerList.length];
+        int i = 0;
+        for (String zkServer : zkServerList) {
+            String[] hostAndPort = StringUtils.split(zkServer, ':');
```

Comment:
Guava already exists and has a HostAndPort.fromString("...").withDefaultPort(2181)